mautic/mautic
# Re: FullCalendar localization

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging for features or enhancements
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | N/A
| Related developer documentation PR URL | N/A
| Issue(s) addressed                     | N/A

#### Description:

Retry to localize fullcalendar. #8251 is the previous pull request but it made for mautic v2.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Go to Configuration page then choose Language
3. Logout from Mautic
3. Go to the Calendar page and see your language

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9432"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
